examples/custom_model_runner/locustfile.py

```diff
@@ -0,0 +1,16 @@
+from locust import task
+from locust import between
+from locust import HttpUser
+
+
+class MnistTestUser(HttpUser):
+    @task
+    def predict(self):
+        url = "/predict"
+        filename = "./mnist_png/testing/9/1000.png"
```

Comment:
would be nice to include the example image here.

Reply:
the README contains instruction for downloading this